debezium/README.md

```diff
@@ -1,11 +1,43 @@
 # Debezium Connector
 
-A Hazelcast Jet connector for [Debezium](https://debezium.io/) which enables 
-Hazelcast Jet pipelines to consume CDC events from various databases.
+[Change data capture (CDC)](https://en.wikipedia.org/wiki/Change_data_capture)
+is a set of software design patterns used to determine (and track) the
+data that has changed so that action can be taken using the changed
+data. CDC is also an approach to data integration that is based on the
+identification, capture and delivery of the changes made to enterprise
+data sources.
+
+[Debezium](https://debezium.io/) is an open source distributed platform
+for change data capture. Start it up, point it at your databases, and
+your apps can start responding to all of the inserts, updates, and
+deletes that other apps commit to your databases. According to the [Debezium website](https://debezium.io/documentation/faq/#what_are_some_uses_of_debezium)
```

Comment:
moving long links to its own line would be better, there are a few more occurrences down below

Reply:
When rendered they don't make any difference and linters are fine with links with more than 72/80 chars.